src/styles/utils/_variables.scss

```diff
@@ -0,0 +1 @@
+$duration: 300ms;
```

Comment:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file